libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/models/FeedbackRecords.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.ai.qna.models;
+
+/**
+ * Active learning feedback records.
+ */
+public class FeedbackRecords {
+    @JsonProperty("feedbackRecords")
+    private String[] records;
```

Comment:
This should be `FeedbackRecord` instead of `String`. See [here](https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Builder.AI.QnA/Models/FeedbackRecords.cs#L21)